umibukela/models.py

```diff
@@ -53,3 +50,62 @@ class Partner(models.Model):
 
     def __str__(self):
         return "[ID: %s] %s" % (self.slug, self.short_name)
+
+
+class Site(models.Model):
+    name = models.CharField(max_length=200)
+    slug = models.CharField(max_length=200, primary_key=True)
```

Comment:
cool. I'll fix

On 14 January 2016 at 11:43, Greg Kempe <notifications@github.com> wrote:

> In umibukela/models.py
> <https://github.com/Code4SA/umibukela/pull/19#discussion_r49704737>:
>
> > @@ -53,3 +50,62 @@ class Partner(models.Model):
> >
> >      def __str__(self):
> >          return "[ID: %s] %s" % (self.slug, self.short_name)
> > +
> > +
> > +class Site(models.Model):
> > +    name = models.CharField(max_length=200)
> > +    slug = models.CharField(max_length=200, primary_key=True)
>
> The slug should be unique, but we should definitely have a regular integer
> primary key. Otherwise referential integrity becomes a mission if the name
> (and hence slug) change.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Code4SA/umibukela/pull/19/files#r49704737>.
>